ansible/roles/test/files/ptftests/copp_tests.py

```diff
@@ -257,6 +258,41 @@ def contruct_packet(self, port_number):
 
         return packet
 
+# SONIC configuration has no packets to CPU for DHCP-T1 Topo
+class DHCPTopoT1Test(PolicyTest):
+    def __init__(self):
+        PolicyTest.__init__(self)
+        # T1 DHCP no packet to packet to CPU so police rate is 0
+        self.PPS_LIMIT_MIN = 0
+        self.PPS_LIMIT_MAX = 0
+
+    def runTest(self):
+        self.log("DHCPTopoT1Test")
+        self.run_suite()
+
+    def contruct_packet(self, port_number):
```

Comment:
Is this packet different from the one from the original DHCP test?

Reply:
no it is same packet. are you thinking  we can inherit from original DHCP Test Class ?

Reply:
It's a NoPolicy test v. Policy test so we probably can't inherit directly, but I think we could probably put it in a shared location!

Reply:
@dall i am thinking to change the packet to Unicast for T1 
T1 will receive unicast DHCP as T0 will be running DHCP-Relay. 
Then we can keep current way per class.

Reply:
That sounds pretty reasonable to me!